demux/demux_lavf.c

```diff
@@ -1005,6 +1006,9 @@ static int demux_open_lavf(demuxer_t *demuxer, enum demux_check check)
                    "analyzeduration to %f\n", analyze_duration);
     }
 
+    if (priv->format_hack.no_ext_picky)
+        av_dict_set(&dopts, "extension_picky", "0", 0);
```

Comment:
suggestion:
```c
if (av_dict_set(...) == /* success */)
    MP_VERBOSE(demuxer, "Option extension_picky=0 set due to known FFmpeg bugs\n");
```

also if the user wants to keep extension_picky=1 can he do that?

Reply:
yes, we set this before user options are applied.

Reply:
Adding a message for this could be technically wrong because if user sets `demuxer-lavf-o=extension_picky=1` to enable it anyway, it isn't known at this time.